src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -101,7 +101,13 @@ public Response listStacks(@PathParam("instanceName") String instanceName) throw
     public Response isAdmin(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
         UserProfile userProfile = UserProfileManager.getUserProfile();
         String token = userProfile.getAccessToken();
-        GitHubClient client = new GitHubClient();
+        String apiUrl = GithubClientHelper.getApiURL(instanceName);
+        GitHubClient client;
+        if("https://api.github.com".equals(apiUrl)){
+            client = new GitHubClient(); 
+        }else{
+            client = new GitHubClient(apiUrl);            
+        }
```

Comment:
all of this should be 1 line `GitHubClient client = GitHubClientInitilizer.getClient()`